packages/core/src/components/button/_common.scss

```diff
@@ -184,6 +188,10 @@ $non-default-state-selectors: ":last-child, :hover, :active, .pt-active, [class*
     background-image: none;
     color: $dark-button-color-disabled;
   }
+
+  .pt-button-spinner .pt-spinner-head {
+    stroke: $pt-dark-text-color;
```

Comment:
@greglo I think that's the one Piotr is talking about. We could use `$dark-spinner-head-color` to be consistent with spinners. Don't worry about it though, I'll take care of that as part of https://github.com/palantir/blueprint/issues/369